services/static-webserver/client/source/class/osparc/desktop/credits/BuyCredits.js

```diff
@@ -225,24 +221,12 @@ qx.Class.define("osparc.desktop.credits.BuyCredits", {
       this.getChildControl("auto-recharge-button");
     },
 
-    __applyTotalPrice: function(totalPrice) {
-      let creditPrice = this.self().CREDIT_PRICES[0][1];
-
-      if (totalPrice >= this.self().CREDIT_PRICES[1][0]) {
-        creditPrice = this.self().CREDIT_PRICES[1][1];
+    __updateNCredits: function() {
+      const totalPrice = this.getTotalPrice();
+      const creditPrice = this.getCreditPrice();
+      if (totalPrice !== null && creditPrice !== null) {
+        this.setNCredits(totalPrice / creditPrice);
```

Comment:
:+1: 
FYI: I noticed "the original OM" ;-) uses in his slides credits/dollar instead of dollar/credit so we might invert the result of the value returned by the API